File: challenges/4-fizz-buzz/elixir/digzom/fizz_buzz.exs
defmodule FizzBuzz do
  def exec(number) when number < 1, do: 0

  def exec(number) do
    1..number
    |> Enum.map(&fizz_buzz/1)
    |> Enum.map(&IO.puts/1)
  end

  def fizz_buzz(el) when Kernel.rem(el, 5) === 0 and Kernel.rem(el, 5) === 0, do: "FizzBuzz"
  def fizz_buzz(el) when Kernel.rem(el, 5) === 0, do: "Buzz"
  def fizz_buzz(el) when Kernel.rem(el, 3) === 0, do: "Fizz"
  def fizz_buzz(el), do: el
end

System.argv()
|> List.first()
|> String.to_integer()
|> FizzBuzz.exec()

File: challenges/4-fizz-buzz/javascript/edurevsky/fizz-buzz.js

const x = process.argv[2]

Array.from({ length: x }, (_, i) => i + 1)
  .map(number => 
    number % 5 == 0 && number % 3 == 0 ? 'FizzBuzz' 
      : number % 5 == 0 ? 'Buzz' 
      : number % 3 == 0 ? 'Fizz' 
      : number
  )
  .forEach(element => console.log(element))
